mypy/stubutil.py

```diff
@@ -87,7 +87,8 @@ def find_unique_signatures(sigs: Sequence[Sig]) -> List[Sig]:
 
 
 def is_c_module(module: ModuleType) -> bool:
-    return '__file__' not in module.__dict__ or module.__dict__['__file__'].endswith('.so')
+    return '__file__' not in module.__dict__ or any(module.__dict__['__file__'].endswith(
+      ext) for ext in ['.so', '.pyd'])
```

Comment:
This is causing the linter to fail:
```
./mypy/stubutil.py:91:7: E121 continuation line under-indented for hanging indent
```
You can run the linter locally: `flake8 mypy/stubutil.py`

Though honestly I would refactor the code a little more -- the `any()` feels like it makes the code more complex than it needs to be. Maybe use `os.path.splitext()`?